src/main/java/Duke.java

```diff
@@ -1,10 +1,218 @@
-public class Duke {
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.stage.Stage;
+
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+
+import javafx.scene.layout.Region;
+
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
+
+import java.util.ArrayList;
+
+
+public class Duke extends Application{
+
+    private Storage storage;
+    private TaskList tasks;
+    private UserInterface ui;
+    private ArrayList<String> userIn = new ArrayList<>();
+    private ArrayList<String> reply = new ArrayList<>();
+    private ScrollPane scrollPane;
+    private VBox dialogContainer;
+    private TextField userInput;
+    private Button sendButton;
+    private Scene scene;
+
+    private Image user = new Image(this.getClass().getResourceAsStream("/images/User.jpg"));
```

Comment:
You can consider storing the image path as a private static string to make it neater.